artifacts/deploy/webhook-configuration.yaml

```diff
@@ -139,6 +139,20 @@ webhooks:
     sideEffects: None
     admissionReviewVersions: ["v1"]
     timeoutSeconds: 10
+  - name: resourceinterpretercustomization.config.karmada.io
```

Comment:
The naming format of the name is different from the current format. Maybe we can adjust the naming format in a unified manner.

Reply:
Suggest renaming it to `resourceinterpretercustomization.karmada.io`.